src/abyssal_tome/app.py

```
@@ -0,0 +1,601 @@
+class EntryType(StrEnum):
+    UNKNOWN = "unknown"
+    ERRATUM = "erratum"
+    QUESTION_ANSWER = "question/answer"
+    CLARIFICATION = "clarification"
+
+@unique
+class QAType(StrEnum):
+    QUESTION = "question"
+    ANSWER = "answer"
+
+TAG_TO_LETTER = {
```

Ruff (0.11.9)

Expected 2 blank lines after class or function definition, found (1)

(E305)